db/migrate/20150413103214_add_solicitor_and_cco_uuids_to_defence_requests.rb

```diff
@@ -0,0 +1,6 @@
+class AddSolicitorAndCcoUuidsToDefenceRequests < ActiveRecord::Migration
+  def change
+    add_column :defence_requests, :solicitor_uuid, :uuid
+    add_column :defence_requests, :cco_uuid, :uuid
```

Comment:
Shall we go with `_uid` as the prefix convention for UID fields, instead of `_uuid` ?

`_uid` feels better given we defined `uid` fields for Profile and Organisation in the [auth app schema](https://github.com/ministryofjustice/defence-request-service-auth/blob/master/db/schema.rb#L72).